app/src/views/TeamView.vue

```diff
@@ -5,8 +5,7 @@
       <TeamCard
         v-for="team in teams"
         :key="team.id"
-        :TeamName="team.name"
-        :TeamDescription="team.description"
+        :team="team"
```

Comment:
I turn the props into an object. 
SO you don't have to update you component if you object change.